packages/gatsby-link/src/__tests__/index.js

```diff
@@ -6,10 +6,10 @@ import {
   createHistory,
   LocationProvider,
 } from "@reach/router"
-import Link, { navigate, push, replace, withPrefix } from "../"
+import Link, { navigate, push, replace, withPrefix, withAssetPrefix } from "../"
 
 afterEach(() => {
-  global.__PATH_PREFIX__ = ``
+  global.__BASE_PATH__ = ``
```

Comment:
__BASE_PATH__ is the new __PATH_PREFIX__. __PATH_PREFIX__ is now overloaded with assetPrefix + pathPrefix.